src/MeetingsSDKAdapter.js

```diff
@@ -354,8 +362,9 @@ export default class MeetingsSDKAdapter extends MeetingsAdapter {
           remoteAudio: null,
           remoteVideo: null,
           remoteShare: null,
-          showRoster: null,
           state: MeetingState.NOT_JOINED,
+          showRoster: null,
```

Comment:
why not just add `showSettings` between `showRoster` and `state` and not move `showRoster`